test/auth_section/sign_in/widget_test.dart

```diff
@@ -49,13 +54,13 @@ class MockFirebaseAuthService extends AuthService {
 
 void main() {
   group('auth tests', () {
-    testWidgets('auth error: shows dialog with error message',
+    testWidgets('navigate to lobby on successfull login',
         (WidgetTester tester) async {
       final navService = NavigationService();
+      final authService = MockFirebaseAuthService(navService, true);
```

Comment:
Personally I think your mocks should use variable names that include 'mock', eg. I reckon `authService` here
 should be `mockAuthService`. It's especially important I would say where you are using both real services and mock services.

Reply:
Yeah good point! Thank you :) It would be good to differentiate those as much as possible.